### DISCUSSION TITLE
AUR package

### DISCUSSION THREAD

**Discussion by junkfactory** [ddterm/gnome-shell-extension-ddterm#74] [General]
Hi, I was about to ask permission from you guys but I accidentally pushed to the remote AUR repository https://aur.archlinux.org/packages/gnome-shell-extension-ddterm. Please let me know if you would like me to transfer ownership to you guys.

**Comment by amezin** [comment]
Look at other extension packages, for example, [appindicator](https://github.com/archlinux/svntogit-community/blob/packages/gnome-shell-extension-appindicator/trunk/PKGBUILD) and [gTile](https://github.com/archlinux/svntogit-community/blob/packages/gnome-shell-extension-gtile/trunk/PKGBUILD)

`gnome-extensions install -f` is not the correct way to build the package. You shouldn't use `gnome-extensions install` in PKGBUILD at all. The package generated by your PKGBUILD is most likely empty.

**Reply by junkfactory** [reply]
Updated

**Reply by amezin** [reply]
I'm an Arch user myself. But I install GNOME Shell extensions using `chrome-gnome-shell`, so I wouldn't want to maintain an AUR package.

So thank you for creating and maintaining it. Just make sure that it actually works :)

Would be nice if there also was a `-git` variant that tracks the current `master` branch.
